pkg/juicefs/mount/pod_mount.go

```diff
@@ -436,6 +453,45 @@ func (p *PodMount) AddRefOfMount(ctx context.Context, target string, podName str
 	return nil
 }
 
+func (p *PodMount) setUUIDAnnotation(ctx context.Context, podName string, uuid string) (err error) {
+	var pod *corev1.Pod
+	pod, err = p.K8sClient.GetPod(context.Background(), podName, jfsConfig.Namespace)
+	if err != nil {
+		return err
+	}
+	klog.Infof("setUUIDAnnotation: set pod %s annotation %s=%s", podName, jfsConfig.JuiceFSUUID, uuid)
+	return util.AddPodAnnotation(ctx, p.K8sClient, pod, map[string]string{jfsConfig.JuiceFSUUID: uuid})
+}
+
+// GetJfsVolUUID get UUID from result of `juicefs status <volumeName>`
+func (p *PodMount) GetJfsVolUUID(ctx context.Context, name string) (string, error) {
+	cmdCtx, cmdCancel := context.WithTimeout(ctx, 8*defaultCheckTimeout)
+	defer cmdCancel()
+	stdout, err := p.Exec.CommandContext(cmdCtx, jfsConfig.CeCliPath, "status", name).CombinedOutput()
+	if err != nil {
+		re := string(stdout)
+		if strings.Contains(re, "database is not formatted") {
+			klog.V(6).Infof("juicefs %s not formatted.", name)
+			return "", nil
```

Comment:
Should it also be an unexpected case? As we already called `waitUtilMountReady` before calling `GetJfsVolUUID`.

Reply:
done